styling-and-themes/swatch-distribution.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Swatch Distribution
+page_title: Swatch Distribution
+description: The UI for Blazor suite distributes most commonly used swatches. Explore how you can access them.
+slug: swatch-distribution
+tags: telerik,blazor,theme,swatch,distribution
+published: True
+position: 2
+---
+
+# Swatch Distribution
+
+Telerik UI for Blazor distributes the most commonly used swatches. While it is possiblle to get them from the [Themes CDN]({%slug themes-swatches%}#buit-in-themes-and-swatches-list), the [Sass Theme Builder]({%slug themes-custom%}#using-the-sass-theme-builder) or the [kendo-themes repository](https://github.com/telerik/kendo-themes), this article lists a couple other ways of distribution.
+
+>caption In this article:
+
+* [Distributed Swatches](#distributed-swatches)
+* [Distribution Options](#distribution-options)
+    * [Telerik Blazor CDN](#telerik-blazor-cdn)
+    * [ZIP archive](#zip-archive)
+    * [VS and VS Code Wizard](#vs-and-vs-code-wizard)
```

Comment:
We shouldn't abbreviate trademarks in official documentation.

Reply:
Fixed